test/string.spec.js

```diff
@@ -0,0 +1,210 @@
+      'Fjzz Bvzz'.hasVowels().should.be.false;
+    });
+  });
+
+  describe('toUpper', () => {
+    it('should return a String', () => {
+      'Hello World'.toUpper().should.be.a('String');
+    });
+    it('should covert each letter to uppercase', () => {
+      'hello world'.toUpper().should.equal('HELLO WORLD')
+    });
+    it('should covert each letter to uppercase for mixed case words', () => {
+      'HEllO andEla'.toUpper().should.equal('HELLO ANDELA')
+    });
+    it('should properly handle symbols', () => {
+      'HE*$?llO andEla'.toUpper().should.equal('HE*$?LLO ANDELA')
+    });
+    it('should not utilize the toUpperCase method', () => {
+      String('HEllO andEla'.toUpper).should.not.have.string('toUpperCase')
+    });
+  });
+
+  describe('toLower', () => {
+    it('should return a String', () => {
+      'HELLO WORLD'.toLower().should.be.a('String')
+    });
+    it('should convert each letter to lowercase', () => {
+      'HELLO WORLD'.toLower().should.equal('hello world')
+    });
+    it('should convert each letter to lowercase for mixed case words"', () => {
+      'heLlO wOrlD'.toLower().should.equal('hello world')
+    });
+    it('should not utilize the toLowerCase method', () => {
+      String('HEllO andEla'.toLower).should.not.have.string('toLowerCase')
+    });
+  });
+
+  describe('ucFirst', () => {
+    it('should return a String', () => {
+      'hELLo WORLD'.ucFirst().should.be.a('String');
+    });
+    it('should convert the first character to upper case', () => {
+      'hello world'.ucFirst().should.equal('Hello world');
+      'heLLo wOrld'.ucFirst().should.equal('HeLLo wOrld');
+    });
+    it('should convert the first word character to upper case', () => {
+      '?hello world'.ucFirst().should.equal('?Hello world');
+      '123heLLo wOrld'.ucFirst().should.equal('123HeLLo wOrld');
+    });
+    it('should utilize the toUpper method', () => {
+      String('hellO andEla'.ucFirst).should.have.string('toUpper')
+    });
+  });
+
+  describe('isQuestion', () => {
+    it('should return a boolean', () => {
+      'Is today a public Holiday?'.isQuestion().should.be.a('boolean');
+    });
+    it('should return true if a string ends with a question mark(?)', () => {
+      'Is today a public Holiday?'.isQuestion().should.be.true;
+    });
+    it('should return true if a string ends with a question mark(?) with trailing whitespaces', () => {
+      'Is today a public Holiday?  '.isQuestion().should.be.true;
+    });
+    it('should return false if a string doesn\'t end with a question mark(?)', () => {
+      'Today is a public Holiday'.isQuestion().should.be.false;
+    });
+    it('should return false if a question mark is present but not at the end', () => {
+      'Is today a public ? Holiday'.isQuestion().should.be.false;
+      'Is today a pub?lic Holiday'.isQuestion().should.be.false;
+    });
+  });
+
+  describe('words', () => {
+    it('should return an Object', () => {
```

Comment:
NITPICK: This test is probably not necessary, since the next one checks that it returns an array. (An array is an object).